tests/behaviour/expectations/behaviour.ts

```diff
@@ -3792,6 +3792,9 @@ export const expectations = flatten(
           Expect.Simple('test', [], []),
           Expect.Simple('returnTest', [], ['12', '0']),
         ]),
+        File.Simple('removeIdentityFunctions', [
+          Expect.Simple('f', ['10', '0'], ['3', '20', '0', '5', '0', '13', '0']),
+        ]),
```

Comment:
Is there a way of testing that function `h` and `g` got pruned correctly. Like calling them and catching the error because they do not exist

Reply:
The thing is for those functions to be pruned they most not be externally visible, so even if there was no pruning at all, calling them should always throw an error. In other words, I think that trying to call those functions won't work as a way of testing this pass.

Reply:
Right, that's true